hello/RelatedArtistsPlaylist.py

```diff
@@ -23,20 +23,167 @@
+                songs.append(x['uri'])                                          # get songs by related artists
+                # print(songs)
+    
+
+
+    def atribecalledquestPlaylist(self):
+        searchResults = spotifyObject.search('A+Tribe+Called+Quest', 1, 0, 'artist')     # search for selected artist object
+        getArtist = searchResults['artists']['items'][0]
+        artistId = getArtist['id']          # get slected artist's Id
+
+        relatedArtists = spotifyObject.artist_related_artists(artistId)
+        artists = relatedArtists['artists']     # search for related artists
+
+        for x in artists:
+            artistName = (x['name'])
+            artistSearch = spotifyObject.search(artistName, 2, 0 ,'track')      # search all related artists
+            tracks = artistSearch['tracks']['items']
+            for x in tracks:
+                songs.append(x['uri'])                                          # get songs by related artists
+                # print(songs)
+    
+
+
+    def travisscottPlaylist(self):
+        searchResults = spotifyObject.search('Travis+Scott', 1, 0, 'artist')     # search for selected artist object
+        getArtist = searchResults['artists']['items'][0]
+        artistId = getArtist['id']          # get slected artist's Id
+
+        relatedArtists = spotifyObject.artist_related_artists(artistId)
+        artists = relatedArtists['artists']     # search for related artists
+
+        for x in artists:
+            artistName = (x['name'])
+            artistSearch = spotifyObject.search(artistName, 2, 0 ,'track')      # search all related artists
+            tracks = artistSearch['tracks']['items']
+            for x in tracks:
+                songs.append(x['uri'])                                          # get songs by related artists
+                # print(songs)
+    
+
+    def timmcgrawPlaylist(self):
+        searchResults = spotifyObject.search('Tim+McGraw', 1, 0, 'artist')     # search for selected artist object
+        getArtist = searchResults['artists']['items'][0]
+        artistId = getArtist['id']          # get slected artist's Id
+
+        relatedArtists = spotifyObject.artist_related_artists(artistId)
+        artists = relatedArtists['artists']     # search for related artists
+
+        for x in artists:
+            artistName = (x['name'])
+            artistSearch = spotifyObject.search(artistName, 2, 0 ,'track')      # search all related artists
+            tracks = artistSearch['tracks']['items']
+            for x in tracks:
+                songs.append(x['uri'])                                          # get songs by related artists
+                # print(songs)
+
+
+    def nancyajramPlaylist(self):
+        searchResults = spotifyObject.search('Nancy+Ajram', 1, 0, 'artist')     # search for selected artist object
+        getArtist = searchResults['artists']['items'][0]
+        artistId = getArtist['id']          # get slected artist's Id
```

Comment:
A list must be filled with as many songs as possible (without sacrificing too much speed) and songs must be randomly selected from that list